include/slate/enums.hh

```diff
@@ -65,6 +74,7 @@ enum class Option : char {
     MaxPanelThreads,    ///< max number of threads for panel, >= 1
     Tolerance,          ///< tolerance for iterative methods, default epsilon
     Target,             ///< computation method (@see Target)
+    Algorithm,          ///< algorithm used in the routine
```

Comment:
We should group this with the other methods below ([lines 95-100](https://github.com/icl-utk-edu/slate/pull/16/files#diff-3934dee5cbdbc3f67f135a65bf92f02358cc21b8fd136629ffcbd070917b4e27R95-R100)) and make it more consistent with those. Maybe `MethodEig` instead of `Algorithm`? Then the enum values could be shortened, perhaps:
```
MethodEig::QR
MethodEig::DC
```
Thoughts?
But I think we can keep with an enum, instead of the class structure that other methods are using.

Reply:
Yes, `MethodEig` looks good and consistent with other algorithm methods.

I moved to methods and here is how its enum values:
```
//------------------------------------------------------------------------------
/// Eigenvalue algorithm to used in heev routine.
/// @ingroup enum
///
enum class MethodEig : char {
    QR        = 'Q',    ///< QR iteration for finding eigenvalues
    DC        = 'D',    ///< Divide and conquer algorithm for finding eigenvalues
};
```